package/samplers/pfns4bo/README.md

```diff
@@ -0,0 +1,67 @@
+---
+author: "HideakiImamura"
+title: "PFNs4BO sampler"
+description: "In-context learning for Bayesian optimization. This sampler uses Prior-data Fitted Networks (PFNs) as a surrogate model for Bayesian optimization."
+tags: ["sampler"]
+optuna_versions: ["3.6.1"]
+license: "MIT License"
+---
+
+## Class or Function Names
+- PFNs4BOSampler
+
+## Installation
+```bash
+pip install -r requirements.txt
```

Comment:
The following will be more helpful because this allows users to install the dependencies by copy and paste. The URL becomes valid after merging this PR.

```suggestion
pip install -r https://raw.githubusercontent.com/optuna/optunahub-registry/main/package/samplers/pfns4bo/requirements.txt
```

Reply:
I think resolving dependencies might fail when the environment of users do not match the latest main branch. It would be great as it is.